frontend/src/components/LandingPage/FeaturedCourses.jsx

```
@@ -0,0 +1,176 @@
+						/>
+					))}
+				</div>
+			</div>
+			{/*Top Rated courses section*/}
+			<h2 className="text-3xl font-bold text-center text-gray-800 mt-16 mb-10">
+				Top Rated Courses
+			</h2>
+			{/* Desktop grid layout */}
+			<div className="hidden md:grid md:grid-cols-3 gap-8">
+				{sortedCourses.slice(0, NoOfCardsToShow).map((course) => (
+					<div className="transform hover:scale-105 hover:shadow-xl transition-all duration-300 cursor-pointer">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)